### PR TITLE
Increase timeouts postgres

### PR DESCRIPTION
**Pull request by mgrossu**
A higher increase of the timeouts.

[CI:TOXENVS] postgres

### CODE REVIEW

File: tests/test_postgres.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def _generate_test_matrix() -> List[ParameterSet]:
                         ),
                         # https://github.com/SUSE/BCI-tests/issues/647
                         healthcheck_timeout=(
-                            timedelta(minutes=3)
+                            timedelta(minutes=4)
```
Comment (grisu48) [inline tests/test_postgres.py:73]: If 3 was not enough, then yank it up to 6. If we are on the brick this will only lead to sporadic failures.

**Comment by grisu48** [comment]
Please add a link to this PR to https://github.com/SUSE/BCI-tests/issues/647#issuecomment-2473548314, so that we can track which PRs we need to revert once we have the actual PPC workers back.

File: bci_tester/data.py [hunk 2]
```diff
@@ -727,6 +727,12 @@ def create_BCI(
         available_versions=pg_versions,
         forwarded_ports=[PortForwarding(container_port=5432)],
         extra_environment_variables={"POSTGRES_PASSWORD": POSTGRES_PASSWORD},
+        # FIXME: https://github.com/SUSE/BCI-tests/issues/647
```
Comment (dirkmueller) [inline bci_tester/data.py:730]: please remove FIXME, that just lowers the pylint scoring and somebody else needs to compensate for that.

**Review by dirkmueller — APPROVED** [review]

File: tests/test_postgres.py (hunk 1 above)
Comment (mgrossu) [reply tests/test_postgres.py]: Increased to 6 minutes.

File: bci_tester/data.py (hunk 2 above)
Comment (mgrossu) [reply bci_tester/data.py]: Removed it.

**Pull request merged**